Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocSimpleColumnRenderer.cs

```diff
@@ -111,18 +111,16 @@ protected override void RenderTitleCell (BocColumnRenderingContext<BocSimpleColu
 
         if (!string.IsNullOrEmpty(boundPropertyPath))
         {
-          renderingContext.Writer.AddAttribute(DiagnosticMetadataAttributesForObjectBinding.HasPropertyPaths, "true");
-          renderingContext.Writer.AddAttribute(
+          renderingContext.AddAttributeToRender(DiagnosticMetadataAttributesForObjectBinding.HasPropertyPaths, "true");
+          renderingContext.AddAttributeToRender(
               DiagnosticMetadataAttributesForObjectBinding.BoundPropertyPaths,
               boundPropertyPath);
         }
         else
         {
-          renderingContext.Writer.AddAttribute(DiagnosticMetadataAttributesForObjectBinding.HasPropertyPaths, "false");
+          renderingContext.AddAttributeToRender(DiagnosticMetadataAttributesForObjectBinding.HasPropertyPaths, "false");
         }
       }
-
-      base.RenderTitleCell(renderingContext, arguments);
```

Comment:
base-calls are mandatory unless the base is abstract or we intentionally supress the base-behavior. Otherwise, we will have a bug sometime in the future when the base contains logic.